__test__/components/comments/CommentItem.test.js

```diff
@@ -17,30 +17,33 @@ const props = {
   isLoading: false
 };
 
-test('It should render the comment item component', () => {
-  const wrapper = shallow(<CommentItem comment={comments[0]} {...props}/>);
-  expect(wrapper).toMatchSnapshot();
-});
```

Comment:
Why did you remove this snapshot?

Reply:
It was failing on Travis.

Reply:
Not a good way of doing things man... at least not without calling my attention to it.